### PR TITLE
[new] signing address validation

### PR DESCRIPTION
**Pull request by GustavoRSSilva**
Added validation, to check if the signature is the same as the sender

### CODE REVIEW

File: test/extendedAdvertisement.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ var privateKey2;
 var objSign0;
```
Comment (DioPires) [inline test/extendedAdvertisement.js:33]: #### Tests:
```diff 
+ 111 passing
```
#### Test coverage: 
82.44 %

Comment (DioPires) [inline test/extendedAdvertisement.js:33]: #### Tests:
```diff 
+ 111 passing
```
#### Test coverage: 
82.44 %

Comment (DioPires) [inline test/extendedAdvertisement.js:33]: #### Tests:
```diff 
+ 111 passing
```
#### Test coverage: 
82.44 %

File: test/signature.js [hunk 2]
```diff
@@ -30,4 +30,18 @@ contract('Signature', function(accounts) {
 
 	})
 
+	it('Check that the send is the one that validated the message', async function () {
```
Comment (tcrosado) [inline test/signature.js]: typo "send" should be "sender"

File: test/signature.js [hunk 3]
```diff
@@ -30,4 +30,18 @@ contract('Signature', function(accounts) {
...
+
+		const privateKey = "0x6b04c8e3bb9969f1455d1ee0d0d22617b84a47d85d0e0fb29498b6e8daa776e6";
```
Comment (tcrosado) [inline test/signature.js]: Should we use address and private key from the ganache-cli configuration we have?

File: test/signature.js (hunk 2 above)
Comment (GustavoRSSilva) [reply test/signature.js]: fixed!

File: test/signature.js (hunk 3 above)
Comment (GustavoRSSilva) [reply test/signature.js]: We can change for that.

**Comment by GustavoRSSilva** [comment]
Closed.

**Pull request closed without merging**